src/QAT/qat/purr/compiler/devices.py

```diff
@@ -234,7 +234,7 @@ def _log_ref(self, obj):
         is_new = super()._log_ref(obj)
         # We don't want complex' or enums to be referenced. Just create a new one on
         # each instance.
-        if isinstance(obj, (complex, Enum)):
+        if isinstance(obj, (complex, Enum, np.number)):
```

Comment:
what does numpy number capture here that complex doesn't? Just curious

Reply:
Numpy wrapped numbers, e.g. `np.float64`